### ISSUE TITLE
Flagged by Virus total as TROJ_GEN.R002V01K123

### ISSUE THREAD

**Issue by rspavaskar** [jtvpremium/jio-jtv-premium#20]
In the TrendMicro-HouseCall scan result on Virustota, it is flagged as TROJ_GEN.R002V01J123. Please clarify and resolve the issue.
<img width="738" alt="Untitled" src="https://github.com/jtvpremium/jio-jtv-premium/assets/123869610/42726efc-c8c7-4a53-87ab-45fb3f994f1a">

**Comment by technofreak24** [comment]
@rspavaskar Yes. Check this screenshot post in the [Discord channel](https://discord.gg/SwMWScYN9D) and message link is https://discord.com/channels/1116342347417002004/1116941091430731826/1179278009371476028
![Screenshot_20240124-141925](https://github.com/jtvpremium/jio-jtv-premium/assets/121749085/1603f5fc-6f0e-4faf-aa4a-ca29cc17588b)

**Comment by techgamers2** [comment]
if its flagged by only one antivirus than its probably a false positive

**Comment by aayushnaphade** [comment]
### it got flagged by my windows antivirus too
scarry!

**Comment by TecnikOfficial** [comment]
can source code of app be made public ?
